README.md

```diff
@@ -0,0 +1,52 @@
+SFML.Net - Simple and Fast Multimedia Library for .Net [![Build Status](https://travis-ci.org/SFML/SFML.Net.svg?branch=master)](https://travis-ci.org/SFML/SFML.Net)
+======================================================
+
+[SFML](sfml-dev.org) is a simple, fast, cross-platform and object-oriented multimedia API. It provides access to windowing,
+graphics, audio and network.
+It is originally written in C++, and this project is its official binding for .Net languages (C#, VB, ...).
+
+Authors
+-------
+
+Laurent Gomila - main developer (laurent@sfml-dev.org)
+Zachariah Brown - active maintainer (contact@zbrown.net)
+
+Download
+--------
+
+You can get the latest official release on [NuGet](https://www.nuget.org/packages/SFML.Net/) or on [the
+SFML website](http://www.sfml-dev.org/download/sfml.net).
+You can also get the current development version from the [git repository](https://github.com/SFML/SFML.Net).
+
+Learn
+-----
+
+There is no tutorial for SFML.Net, but since it's a binding you can use the C++ resources:
+* [The official tutorials](http://www.sfml-dev.org/tutorials/)
+* [The online API documentation](http://www.sfml-dev.org/documentation/)
+* [The community wiki](https://github.com/SFML/SFML/wiki/)
+* [The community forum](http://en.sfml-dev.org/forums/) (or [for French people](http://fr.sfml-dev.org/forums/))
```

Comment:
These should use the HTTPS links

Also, could you change `for French people` to `for French speaking people`?